prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -1249,6 +1254,7 @@ class Hl7Serializer(
         terser: Terser
     ): String {
         val maxLength = getMaxLength(hl7Field, value, hl7Config, terser)
+        println("This is the length: " + maxLength)
```

Comment:
@jorg3lopez we shouldn't have the serializer printing out the length like this in production code. Please remove.

Reply:
Good catch. This was used for testing purposes, I'll remove it asap.